templates/jeopardy.html

```diff
@@ -584,7 +584,7 @@
 </table>
 
 <div id="options">
-    <h1>Science Masters Jeopardy</h1>
+    <h1>Team Aaiaa Sports Jeopardy</h1>
```

Comment:
Changed the title of our Jeopardy page as Science Masters is the internet source that we got the code from.